lib/element.rb

```diff
@@ -6,7 +6,7 @@ class Element
   attr_reader :name, :by, :locator
 
 
-  def initialize(name, by, locator)
+  def initialize(name, by, locator, parent=nil)
```

Comment:
@yet-another-lucas if you wanted to not have `parent=nil` in the constructor, you could add `opts = {}` as the last constructor parameter. Makes things more clean, and explicitly lets a reader of the `Gridium` docs see that optional params are accepted.

`Element.new("Toasty Element", :css, ".some-css", parent: parent_element)`